Extensions/ExtendedVariables.json

```diff
@@ -308,7 +324,8 @@
       "description": "Modify the value of a scene variable.",
       "fullName": "Value of a scene variable",
       "functionType": "Action",
-      "name": "A_Modify_scene_variable",
+      "group": "Scene variables",
+      "name": "ModifySceneVariable",
       "private": false,
       "sentence": "Change the scene variable _PARAM1_ with value: _PARAM2_",
```

Comment:
What about: `Change the text of global variable _PARAM1_: _PARAM2_` to use the same sentence as the built-in action.